src/crypto/chacha.c

```diff
@@ -168,3 +168,13 @@ void chacha8(const void* data, size_t length, const uint8_t* key, const uint8_t*
     data = (uint8_t*)data + 64;
   }
 }
+
+void chacha8(const void* data, size_t length, const uint8_t* key, const uint8_t* iv, char* cipher)
```

Comment:
Is it worth having a unit test for these to check that `chacha(8, ...) == chacha8(...)`?

Reply:
chacha is file static (since it relies on rounds being even), no return code, and C so no exceptions.

Reply:
ok, sgtm